tests/helpers/editor-helpers.js

```diff
@@ -29,6 +29,29 @@ export function titleRendered() {
     });
 }
 
+// replaces the title text content with HTML and returns once the HTML has been placed.
+// takes into account converting to plaintext.
+export function replaceTitleHTML(HTML) {
```

Comment:
We could also generalise this a bit so it can be used with other elements, e.g. `fillInHTML('#gh-editor-title div', '<div>pasted HTML</div>');